src/App.tsx

```diff
@@ -27,15 +28,17 @@ const App = () => {
 
   return (
     <SafeAreaProvider>
-      <RNP.Provider>
-        <AuthProvider>
-          <SelectedWalletMetaProvider>
-            <SelectedWalletProvider>
-              <AppNavigator />
-            </SelectedWalletProvider>
-          </SelectedWalletMetaProvider>
-        </AuthProvider>
-      </RNP.Provider>
+      <SearchProvider>
```

Comment:
move search provider as close to consumer as possible

Reply:
each search context should scoped, not reused
if we need more searching, then add multiple providers
searching on one screen shouldnt affect results on another screen